content/300-guides/200-deployment/110-deployment-guides/450-deploying-to-cloudflare-workers.mdx

```diff
@@ -1,329 +1,75 @@
-
-2. Click **New Project** and then select **Import a Prisma repository**.
-3. Fill in the repository and project details, and click **Next**.
-
-   ![Import a Project](./450-03-import-project.png)
-
-4. Next, you connect the Prisma Data Platform to the MongoDB Atlas database and set up the Data Proxy.
-
-   1. Paste the MongoDB Atlas connection string which also includes the database name as described in [1. Get a MongoDB Atlas connection string](#1-get-a-mongodb-atlas-connection-string).
-   2. Under **Location**, select a Data Proxy location that is geographically close to your MongoDB database location.
-   3. Enable **Static IPs** and copy and paste them in the _Network Access_ screen in MongoDB Atlas.
-
-      ![Connect your Database](./450-04-connect-db.png)
-
-   4. Click **Create Project** to test the connection and set up the Data Proxy.
-
-      <Admonition type="warning">
-
-      If you see, "The database needs to be empty to proceed", you can simply use a different database name. Using the screenshot above, that would be renaming `Log` to something else.
-
-      </Admonition>
-
-      If all goes well, you'll be greeted with a new connection string that starts with `prisma://`.
-
-   5. Copy the `prisma://` connection string to your clipboard.
-
-      ![Data Proxy Page](./450-05-data-proxy.png)
-
-## 7. Set the Data Proxy Connection string in your environment
-
-1. Add the Data Proxy connection string to your local environment `.env` file.
-
-   ```diff file=.env
-   - DATABASE_URL = "mongodb+srv://<user>:<password>@cluster.mongodb.net/Log?retryWrites=true&w=majority"
-   + DATABASE_URL = "prisma://aws-us-east-1.prisma-data.com/?api_key=•••••••••••••••••"
-   ```
-
-2. In your code editor, add the connection string in the `wrangler.toml` file.
-
-   ```diff file=wrangler.toml
-     name = "prisma-mongodb-cloudflare"
-     main = "src/main.ts"
-     compatibility_date = "2022-11-07"
+1. Update your environment variables in `wrangler.toml` by adding the `DIRECT_DATABASE_URL` variable
 
+```diff file="wrangler.toml"
    + [vars]
-   + DATABASE_URL = "prisma://aws-us-east-1.prisma-data.com/?api_key=•••••••••••••••••"
-   ```
-
-You are now ready to generate a Prisma Client.
-
-## 8. Generate a Prisma Client
-
-Next, you'll generate a Prisma Client that connects through the [Data Proxy](/data-platform/data-proxy) over HTTP.
-
-```terminal
-npx prisma generate --data-proxy
+ + DIRECT_DATABASE_URL = "prisma://accelerate.prisma-data.net/?api_key=•••••••••••••••••"
 ```
 
-The generated Client has a smaller bundle size and is optimized for edge environments like Cloudflare Workers.
+1. Generate Prisma Client that connects through Accelerate over HTTP:
 
-The smaller bundle size is due to the fact that the interfaces talking to the database (the [Prisma engines](/concepts/components/prisma-engines)) are no longer bundled with Prisma Client as this logic is now handled by the Data Proxy.
+   ```terminal
+   npx prisma generate --data-proxy
```

Comment:
For versions above or equal to 5.0.0, you can use the following flag to generate the client  for Accelerate

```suggestion
   npx prisma generate --accelerate
```